src/components/Button/Button.tsx

```diff
@@ -30,38 +33,63 @@ const getContainerColorStyles = (
       background: theme.palette.primary.main,
       color: theme.palette.white,
       boxShadow: "0px 0px 16px #0b82f466",
+      border: "none",
     },
     hover: {
       background: theme.palette.primary.dark,
+      border: "none",
     },
     active: {
       background: theme.palette.primary.deepDark,
+      border: "none",
+    },
+  },
+  secondary: {
+    normal: {
+      background: colors.blue[50] as string,
+      color: theme.palette.text.primary,
+      boxShadow: "none",
+      border: `1px ${colors.blue[500]}`,
```

Comment:
```suggestion
      border: `${Size.Border.Small} solid ${colors.blue[500]}`,
```
borderのサイズは定数あるのでそれを使ってもらえると！

https://developer.mozilla.org/ja/docs/Web/CSS/border
borderの種類も明記してもらえると！

Reply:
わかりました！